File: Livros base/fundamentos da programação 3ª edicao/lista de exercicios - condicionais/ERCondicionais/src/ercondicionais/ER1.java
package ercondicionais;
import java.util.*;

public class ER1 {
    public int trabalho, provaSem, exameFinal, media;
    Scanner input = new Scanner(System.in);

    public int getTrabalho() {
        return trabalho;
    }

    public void setTrabalho(int trabalho) {
        this.trabalho = trabalho;
    }

    public int getProvaSem() {
        return provaSem;
    }

    public void setProvaSem(int provaSem) {
        this.provaSem = provaSem;
    }

    public int getExameFinal() {
        return exameFinal;
    }

    public void setExameFinal(int exameFinal) {
        this.exameFinal = exameFinal;
    }
    
    public void Notas(){
        System.out.println("Escreva a nota do trabalho: ");
        trabalho = input.nextInt();
        System.out.println("Escreva  a nota da prova: ");
        provaSem = input.nextInt();
        System.out.println("Escreva a nota do exame final: ");
        exameFinal = input.nextInt();
        media = (trabalho * 2 + provaSem * 3 + exameFinal * 5) / 10;
        
        if(media >= 8 && media <=10){
            System.out.println("media A");
        }
        if(media >= 7 && media <=8){
            System.out.println("media B");
        }
        if(media >= 6 && media <=7){
            System.out.println("media C");
        }
        if(media >= 5 && media <= 6){
            System.out.println("media D");
        }
        if(media >= 0 && media <=5){
            System.out.println("media E");
        }
    }
}

File: Livros base/fundamentos da programação 3ª edicao/lista de exercicios - condicionais/ER-1.c
#include <stdio.h>

int mediaPonderada(int t, int a, int e);

int mediaPonderada(int t, int a, int e){
	return t*2 + a*3 + e*5 / 10; 
}

void main(){
	int trabalho, a_semestral, exame, mediaP; //notas
	
	printf("Escreva a nota do trabalho: ");
	scanf("%d*c",&trabalho);
	
	printf("\nEscreva a nota da prova semestal: ");
	scanf("%d*c",&a_semestral);
	
	printf("\nEscreva a nota do exame: ");
	scanf("%d*c",&exame);
	
	mediaP = mediaPonderada(trabalho, a_semestral, exame);
	
	if(mediaP >= 8 || mediaP == 10){
		printf("\nConceito A");
	}if(mediaP >= 7 && mediaP <8){
		printf("\nConceito B");
	}if(mediaP >= 6 && mediaP <7){
		printf("\nConceito C");
	}if(mediaP >= 5 && mediaP <6){
		printf("\nConceito D");
	}else if(mediaP >= 0 && mediaP <5){
		printf("\nConceito E");
	}
	
}


